cloud/blockstore/tools/csi_driver/internal/driver/node_test.go

```diff
@@ -411,3 +412,111 @@ func TestPublishUnpublishDeviceForInfrakuber(t *testing.T) {
 	})
 	require.NoError(t, err)
 }
+
+func TestGetVolumeStatCapabilitiesInVmMode(t *testing.T) {
+	tempDir := os.TempDir()
+
+	nbsClient := mocks.NewNbsClientMock()
+	mounter := mounter.NewMock()
+
+	podID := "test-pod-id-13"
+	diskID := "test-disk-id-42"
+	socketsDir := filepath.Join(tempDir, "sockets")
+	targetPath := filepath.Join(tempDir, "pods", podID, "volumes", diskID, "mount")
+	targetFsPathPattern := filepath.Join(tempDir, "pods/([a-z0-9-]+)/volumes/([a-z0-9-]+)/mount")
+
+	nodeService := newNodeService(
+		"testNodeId",
+		"testClientId",
+		true, // vmMode
+		socketsDir,
+		targetFsPathPattern,
+		"", // targetBlkPathPattern
+		nbsClient,
+		nil, // nfsClient
+		mounter,
+	)
+
+	ctx := context.Background()
+	resp, err := nodeService.NodeGetCapabilities(
+		ctx,
+		&csi.NodeGetCapabilitiesRequest{})
+	assert.Nil(t, err)
```

Comment:
Since the csi driver is built using the same ya.make as disk-manager, consider using testify/require.
This line would look: require.NoError(t, err)

Reply:
done